client/components/track-interactions/index.js

```diff
@@ -68,3 +58,17 @@ export default class TrackInteractions extends Component {
 		return cloneElement( child, props );
 	}
 }
+
+function trackInteraction( componentName, data ) {
```

Comment:
Walks like an action, talks like an action -- should live in `state/something/actions` IMO 😄